docs/using/event-types.md

```diff
@@ -47,81 +47,90 @@ have a more specific definition :)
 ### Compatibility modes
 
 Compatibility modes are used to control schema changes. Each mode
-solves a specific problem and thus presents different constraints. It
-ranges from being very constraining, allowing just some minor changes,
-to allowing everything.
+solves a specific problem and thus presents different constraints.
 
-By default, Nakadi uses the forward mode, which is further explained
-below, together with the two other modes: compatible and none.
+Each compatibility mode supports a different set of changes, which are
+described in detail bellow.
 
-#### Forward
+#### Fully compatible
 
-This mode has been designed to allow event type owners to change their
-schemas without breaking existing consumers.
+The `compatible` compatibility mode is the safest mode. It's both
+forward compatible and backward compatible. It means that:
 
-It's called forward because using older schemas, it's safe to read
-events generated by newer schemas. On the other hand, it's not
-backward compatible, in the sense that trying to read an older event
-using a newer schema is not safe.
+1. Consumers using older schemas can safely read events validated by
+   newer schemas.
+2. Clients using newer schemas can safely read events validated by
+   older schemas.
 
-For example, if the consumer has been developed with schema version
-`1.5.0` in mind, reading events from `1.5.0` on is safe, for example,
-`1.6.0`. But reading event generated by older schemas, like `1.4.0`,
-is not safe.
+It guarantees high data quality which is crucial for a variety of
```

Comment:
editorial proposal:
As a consumer, I can pick any schema version that is sufficient for my purpose and Nakadi guarantees that the attributes are explicitly specified and all, i.e. new and old events are compliant to this scheme. Hence, it provides high data quality....